python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,333 @@
+                            await self.__subscribe_to(
+                                csc, salindex, stream, "telemetry"
+                            )
+                    await self.__handle_message_reception()
+        except Exception as e:
+            raise RuntimeError("Manager Client connection failed.") from e
+
+    def create_start_task(self):
+        self.start_task = asyncio.create_task(self.start_ws_client())
+
+    async def close(self):
+        if self.__websocket:
+            await self.__websocket.close()
+
+
+class StressLOVE(salobj.BaseScript):
+    """Run a stress test for one or more CSCs.
+
+    Notes
+    -----
+    **Details**
+
+    * Run a LOVE stress test by generating several client connections
+    that will listen to every event and telemetry of the specified CSCs
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Run a stress test for one or more CSCs")
+
+        # SimpleNamespace to store stress test configurations
+        # params described on `get_schema`
+        self.config = None
+
+        # list to store clients connections,
+        # each one an instance of ManagerClient
+        self.clients = []
+
+        # dict to store remote connections,
+        # with each item in the form of
+        # `CSC_name[:index]`: `lsst.ts.salobj.Remote`
+        self.remotes = {}
+
+        # commands timeout
+        self.cmd_timeout = 10
+
+        # time to wait for each message collection
+        self.loop_time_message_collection = 1
+
+        # message frequency
+        self.expected_message_frequency = 100
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_standardscripts/StressLOVE.yaml
+            title: StressLOVE v1
+            description: Configuration for StressLOVE
+            type: object
+            properties:
+              location:
+                description: Domain of the running LOVE instance to stress
+                type: string
+              number_of_clients:
+                description: The number of clients to create
+                type: number
+              number_of_messages:
+                description: The number of messages to store before calculating the mean latency
+                type: number
+              data:
+                description: List of CSC_name[:index]
+                type: array
+                minItems: 1
+                items:
+                    type: string
+            required: [location, number_of_clients, number_of_messages, data]
+            additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    def set_metadata(self, metadata):
+        """Compute estimated duration.
+
+        Parameters
+        ----------
+        metadata : SAPY_Script.Script_logevent_metadata
```

Comment:
The term `metadata` sounds like data about a topic, rather than message data. I gently suggest `data`.

Also that data type is confusing because SALPY is no longer supported. I suggest something like this (BaseMsgType is especially helpful for type annotations, but it's fine in doc strings as well):
```
data : `lsst.ts.salobj.BaseMsgType`
     Script ``metadata`` event data.
```
All that said: I don't see this method being called. Can you delete it? If I missed where you are calling it, and you are calling it in one place I'd suggest just moving the one line of code there.

Reply:
I thought this was used by `BaseScript` and one had to override it with the corresponding information. Can you confirm if I'm right? I brought this from https://github.com/lsst-ts/ts_standardscripts/blob/develop/python/lsst/ts/standardscripts/set_summary_state.py#L166

Reply:
You are right. My mistake. Thank you.